Partially reverts:
------------------------------------------------------------
revno: 201 [merge]
author: Mathieu Trudel-Lapierre <mathieu-tl@ubuntu.com>, Charles Kerr <charles.k
err@canonical.com>
committer: Tarmac
branch nick: trunk.13.04
timestamp: Wed 2013-01-23 18:04:43 +0000
message:
  Update paths for gtest source files.
  
  Approved by Allan LeSage.
------------------------------------------------------------
=== modified file 'configure.ac'
--- configure.ac	2013-01-17 21:17:40 +0000
+++ configure.ac	2012-11-16 19:51:26 +0000
@@ -58,7 +58,7 @@
 ICAL_REQUIRED_VERSION=0.48
 CAIRO_REQUIRED_VERSION=1.10
 GDK_REQUIRED_VERSION=2.22
-GLIB_REQUIRED_VERSION=2.35.4
+GLIB_REQUIRED_VERSION=2.29.19
 GTK3_REQUIRED_VERSION=3.1.4
 
 PKG_CHECK_MODULES(INDICATOR, indicator3-0.4 >= $INDICATOR_REQUIRED_VERSION

=== modified file 'src/datetime-service.c'
--- src/datetime-service.c	2013-01-17 21:17:40 +0000
+++ src/datetime-service.c	2012-11-16 19:51:26 +0000
@@ -1401,6 +1401,8 @@
 int
 main (int argc, char ** argv)
 {
+	g_type_init();
+
 	/* Acknowledging the service init and setting up the interface */
 	service = indicator_service_new_version(SERVICE_NAME, SERVICE_VERSION);
 	g_signal_connect(service, INDICATOR_SERVICE_SIGNAL_SHUTDOWN, G_CALLBACK(service_shutdown), NULL);

#=== modified file 'tests/Makefile.am'
#--- tests/Makefile.am	2013-01-22 21:35:02 +0000
#+++ tests/Makefile.am	2012-11-07 00:28:32 +0000
#@@ -12,8 +12,8 @@
# 
# check_LIBRARIES = libgtest.a
# nodist_libgtest_a_SOURCES = \
#-  $(GTEST_SOURCE)/gtest-all.cc \
#-  $(GTEST_SOURCE)/gtest_main.cc
#+  $(GTEST_SOURCE)/src/gtest-all.cc \
#+  $(GTEST_SOURCE)/src/gtest_main.cc
# 
# AM_CPPFLAGS = $(GTEST_CPPFLAGS) -I${top_srcdir}/src -Wall -Werror
# AM_CXXFLAGS = $(GTEST_CXXFLAGS)

=== modified file 'tests/test-indicator.cc'
--- tests/test-indicator.cc	2013-01-17 21:17:40 +0000
+++ tests/test-indicator.cc	2012-11-07 00:28:32 +0000
@@ -34,6 +34,7 @@
     if (G_UNLIKELY(!initialized))
     {
       initialized = true;
+      g_type_init();
       g_setenv ("GSETTINGS_SCHEMA_DIR", SCHEMA_DIR, TRUE);
     }
   }

